homeassistant/components/tuya/strings.json

```
@@ -893,6 +896,11 @@
       "siren": {
         "name": "Siren"
       }
+    },
+    "valve": {
+      "indexed_switch": {
+        "name": "Switch {index}"
```

I think it would make more sense to use "Valve" over "Switch".
```suggestion
      "indexed_valve": {
        "name": "Valve {index}"
```